archai/nlp/datasets/nvidia/tokenizer_utils/bbpe_vocab.py

```diff
@@ -133,13 +133,21 @@ def load(self) -> None:
         logger.debug(f"Tokenizer file path: {self._tokenizer_filepath}")
 
     @overrides
-    def encode_text(self, text: str) -> List[int]:
-        text = self._preprocess_text(text)
+    def encode_text(self, text: Union[List, str]) -> List[int]:
+        
```

Comment:
It is just a nitpick, but do you mind removing these extra lines (L137 and L146)? :D